api/src/controllers/generateCv.js

```diff
@@ -0,0 +1,49 @@
+const generateCv = (req, res) => {
+  const {
+    fullName,
+    email,
+    phone,
+    github,
+    linkedin,
+    portfolio,
+    professional_summary,
+    transferable_experience = [],
+    projects = [],
+    education = [],
+    yourProfile_vs_jobCriteria,
+  } = req.body;
+
+  if (
+    !fullName ||
+    !email ||
+    !phone ||
+    !github ||
+    !portfolio ||
+    !linkedin ||
+    !professional_summary ||
+    !transferable_experience ||
+    !projects ||
+    !education ||
+    !yourProfile_vs_jobCriteria
+  ) {
+    return res.status(400).json({ msg: 'All fields are required' });
+  }
```

Comment:
we should have more validation here than whether these fields exist, id suggest looking into [yup](https://www.wisp.blog/blog/validating-api-response-with-yup) and making a schema to validate against